Cargo.lock

```diff
@@ -1361,8 +1366,9 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 [[package]]
 name = "heapsize"
 version = "0.4.2"
-source = "registry+https://github.com/rust-lang/crates.io-index"
+source = "git+https://github.com/cheme/heapsize.git?branch=ec-macfix#aa0fa65a207ad8d66f94c7aec510b2f9dc984dbe"
```

Comment:
Here it is the wrong commit (in merged  it is 421df390a930cb523a09e5528e6fe57b534b3b26 ), we need to force it. 
I am not sure how it update, but I should not have let the latest commit of my branch on a bad state (it uses jemalloc linux and was failling on building some proc_macro (when building registrar) in my tests).

Reply:
Seems like it is passing on the Ci, but it is because the ci is using rust 1.31 (thanks @ordian for noticing).

Reply:
Uh, ok? Well spotted, how to force a certain commit here?